libs/filabridge/src/SamplerInterfaceBlock.cpp

```diff
@@ -76,7 +76,7 @@ SamplerInterfaceBlock::Builder::~Builder() noexcept = default;
 
 SamplerInterfaceBlock::SamplerInterfaceBlock() = default;
 SamplerInterfaceBlock::SamplerInterfaceBlock(const SamplerInterfaceBlock& rhs) = default;
-SamplerInterfaceBlock::SamplerInterfaceBlock(SamplerInterfaceBlock&& rhs) noexcept /* = default */ {};
+SamplerInterfaceBlock::SamplerInterfaceBlock(SamplerInterfaceBlock&& rhs) noexcept = default;
```

Comment:
Doesn't look like it ever gets called, but not sure why this was commented out.

Reply:
That would be a good question to ask @pixelflinger  :)

Reply:
Looks like it succeeds now :)